src/conic_algorithm.jl

```diff
@@ -106,12 +108,12 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
 
     # SOC data
     num_soc::Int                # Number of SOCs
-    t_idx_soc_subp::Vector{Int} # Row index of t variable in SOCs in subproblems
-    v_idxs_soc_subp::Vector{Vector{Int}} # Row indices of v variables in SOCs in subproblem
-    t_soc::Vector{JuMP.AffExpr} # t variable (epigraph) in SOCs
-    v_soc::Vector{Vector{JuMP.AffExpr}} # v variables in SOCs
-    d_soc::Vector{Vector{JuMP.Variable}} # d variables (disaggregated) in SOCs
-    a_soc::Vector{Vector{JuMP.Variable}} # a variables (absolute values) in SOCs
+    r_idx_soc_subp::Vector{Int} # Row index of r variable in SOCs in subproblems
+    t_idx_soc_subp::Vector{Vector{Int}} # Row indices of t variables in SOCs in subproblem
+    r_soc::Vector{JuMP.AffExpr} # r variable (epigraph) in SOCs
+    t_soc::Vector{Vector{JuMP.AffExpr}} # t variables in SOCs
+    pi_soc::Vector{Vector{JuMP.Variable}} # pi variables (disaggregated) in SOCs
```

Comment:
no not for now, sometimes it doesn't display nicely in some editors (eg not fixed width)